kolena/_experimental/object_detection/utils.py

```diff
@@ -267,3 +267,40 @@ def compute_optimal_f1_threshold_multiclass(
     for label, (y_true, y_score) in y_true_score_by_label.items():
         optimal_thresholds[label] = _compute_optimal_f1_with_arrays(y_true, y_score)
     return optimal_thresholds
+
+
+def compute_ap(precisions: List[float], recalls: List[float]) -> float:
+    """
+    Computes the average precision given a PR curve with the metrics methodology of Pascal VOC.
+    Based on https://github.com/Cartucho/mAP which implements the Matlab Pascal Voc code in Python.
```

Comment:
Formatting can be improved — these docstrings are now just Markdown. Also ensure that capitalization is consistent, [elsewhere](https://github.com/kolenaIO/kolena/blob/trunk/kolena/workflow/metrics/_geometry.py#L184) we refer to this as `PASCAL VOC`.